inbm/dispatcher-agent/dispatcher/sota/os_factory.py

```diff
@@ -216,7 +219,7 @@ def create_snapshotter(self, sota_cmd: str, snap_num: Optional[str],
         return DebianBasedSnapshot(trtl, sota_cmd, self._dispatcher_broker,
                                    snap_num, proceed_without_rollback, reboot_device)
 
-    def create_downloader(self) -> Downloader:
+    def create_downloader(self, signature: Optional[str] = None) -> Downloader:
```

Comment:
Document that signature is ignored in doc string.

Reply:
Added.